courses/templates/mail/course_run_enrollment/body.html

```diff
@@ -9,7 +9,7 @@
                   <td style="padding: 20px; font-family: sans-serif; font-size: 15px; line-height: 20px; color: #555555;">
                       <p style="margin: 0 0 10px;">Dear {{ user.name }},</p>
                       <p style="margin: 0 0 10px;">
-                        You have been enrolled in {{ enrollment.run }}. The course should now appear on your <a href="{{ base_url }}{% url 'user-dashboard' %}">{{ site_name }} dashboard</a>.
+                        You have been enrolled in {{ course_number }} {{ enrollment.run.course.title }}. The course should now appear on your <a href="{{ base_url }}{% url 'user-dashboard' %}">{{ site_name }} dashboard</a>.
```

Comment:
As per my above comment:

```suggestion
                        You have been enrolled in {{ enrollment.run.course_number }} {{ enrollment.run.course.title }}. The course should now appear on your <a href="{{ base_url }}{% url 'user-dashboard' %}">{{ site_name }} dashboard</a>.
```